src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -180,6 +190,36 @@ public void MetricObserverCallbackError(string metricName, string exception)
             this.WriteEvent(16, metricName, exception);
         }
 
+        [Event(17, Message = "Batcher finished collection with '{0}' metrics.", Level = EventLevel.Informational)]
+        public void BatcherCollectionCompleted(int count)
+        {
+            this.WriteEvent(17, count);
+        }
+
+        [Event(18, Message = "Collection completed in '{0}' msecs.", Level = EventLevel.Informational)]
+        public void CollectionCompleted(long msec)
+        {
+            this.WriteEvent(18, msec);
+        }
+
+        [Event(19, Message = "Exception occurred in Metric Controller while processing metrics from one Collect cycle. Exception: '{0}'", Level = EventLevel.Warning)]
```

Comment:
this warning is better. The best error message would also give some information on what will not happen because of this warning.